swupd/manifest_test.go

```diff
@@ -210,10 +210,10 @@ func TestCheckInvalidManifestHeaders(t *testing.T) {
 		name   string
 		header ManifestHeader
 	}{
-		{"format not set", ManifestHeader{0, 100, 90, 553, time.Unix(1000, 0), 100000, nil}},
-		{"version zero", ManifestHeader{10, 0, 90, 553, time.Unix(1000, 0), 100000, nil}},
-		{"no files", ManifestHeader{10, 100, 90, 0, time.Unix(1000, 0), 100000, nil}},
-		{"no timestamp", ManifestHeader{10, 100, 90, 553, zeroTime, 100000, nil}},
+		{"format not set", ManifestHeader{0, 100, 90, 0, 553, time.Unix(1000, 0), 100000, nil}},
+		{"version zero", ManifestHeader{10, 0, 90, 0, 553, time.Unix(1000, 0), 100000, nil}},
+		{"no files", ManifestHeader{10, 100, 90, 0, 0, time.Unix(1000, 0), 100000, nil}},
+		{"no timestamp", ManifestHeader{10, 100, 0, 90, 553, zeroTime, 100000, nil}},
```

Comment:
These changes should appear in the commit after this one, because you don't change the `ManifestHeader` struct until then.

Reply:
I ordered the commits to the tests came first... Instead of dealing with this I just reordered the commits so the tests come after the implementation. `¯\_(ツ)_/¯`

Reply:
I'm okay with having test commits come first, but (ideally) mixer-tools should build successfully on every commit. That's what my comment was aimed at.

Reply:
Yeah I agree, I'm just saying the way I resolved your comment is to just reorder the commits instead of picking that bit of history into another commit.

Also, mixer *does* build even with the change that way. Go will not build the test files unless it runs the tests.

Reply:
Oh, right, sorry about not noticing that.